README.md

```diff
@@ -265,6 +265,27 @@ replicator.start();
 
 Read more in [the replication docs](https://github.com/cloudant/sync-android/blob/master/doc/replication.md).
 
+#### Authentication
+
+Sync-android uses session cookies by default to authenticate with the server if
+credentials are provided with in the URL. If you need more fine grain control
```

Comment:
Fine-grained control and Basic Auth are different concepts. How about:
"If you want more fine-grained control over authentication, provide an interceptor to perform the authentication for each request. For example, if you are using a proxy..."